lib/point.js

```diff
@@ -142,12 +142,12 @@ Point.prototype.multiply = function(k) {
   var R = this
 
   for (var i = h.bitLength() - 2; i > 0; --i) {
-    R = R.twice()
-
     var hBit = h.testBit(i)
     var eBit = e.testBit(i)
 
-    if (hBit != eBit) {
+    R = R.twice()
```

Comment:
Putting the `R` calculation here gives clearer visibility that the `testBit` operations done above are *separate*, and non-reliant on it.